src/components/Navbar/ResponsiveAppBar.js

```diff
@@ -13,8 +13,8 @@ import Tooltip from "@mui/material/Tooltip";
 import MenuItem from "@mui/material/MenuItem";
 import { NavLink } from "react-router-dom";
 
-const pages = ["Dashboard", "Search", "Favorite"];
-const settings = ["Profile", "History", "Logout"];
+const pages = ["dashboard", "search", "favorite"];
+const settings = ["profile", "history"];
```

Comment:
These two options, profile history, look lowercase. Should we make them an upper case or capital?
<img width="899" alt="Screen Shot 2022-11-15 at 8 13 53 PM" src="https://user-images.githubusercontent.com/88150284/202082608-43c431d2-85a7-4668-842e-9f3a6552a5fa.png">

Reply:
Yes, they were in upper case, then they appear in upper case in the URL path also.

Reply:
There is no history route. We should not keep things if they are not yet ready for the users. Also I dont see the logout in the menu. So I may convert this PR to draft till your done working on all of it. Also where it says MyKitchen, is it staying like that or its going to be your logo there?

Reply:
for display you can have it Capitalized but in the link it will pass the value which is lowercase